### PR TITLE
Added section in migration guide for registering with PushKit

### PR DESCRIPTION
**Pull request by piyushtank**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by bobiechen-twilio — CHANGES_REQUESTED** [review]
I think this paragraph deserves equally emphasized as performing registration with the new SDK when app is launched.

**Comment by piyushtank** [comment]
I like your suggestions @bchen-twilio, lets get feedback from everyone then I will make the changes.

File: iOS-13-Migration-Guide.md [hunk 1]
```diff
@@ -454,3 +454,7 @@ If your App supports incoming calls, you **MUST** perform the following steps to
 You can reference the 2.1 quickstart for [obj-c](https://github.com/twilio/voice-quickstart-objc/tree/2.x) and [swift](https://github.com/twilio/voice-quickstart-swift/tree/2.x) when migrating your application.
 
 A summary of the API changes and new Insights events can be found in the [2.1.0 changelog](https://www.twilio.com/docs/voice/voip-sdk/ios/2x-changelog#210-september-5-2019).
+
+
+## Register Your App with PushKit at Launch Time
```
Comment (idelgado) [inline iOS-13-Migration-Guide.md]: The wording is fine but I would move this as a step in each migration guide. Finding at the bottom was surprising to me and hard to find.

**Review by idelgado — CHANGES_REQUESTED** [review]

**Review by aymenn — APPROVED** [review]

File: iOS-13-Migration-Guide.md (hunk 1 above)
Comment (piyushtank) [reply iOS-13-Migration-Guide.md]: Yeah, @bchen-twilio had the same comment. I have moved the section to top of the migration guide. WDYT @idelgado ?

File: iOS-13-Migration-Guide.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ iOS 13 introduced changes in push notifications handling. This document describe
 - [iOS 13 & Xcode 10 or below](#ios-13--xcode-10-or-below)
 - [iOS 13 & Xcode 11](#ios-13--xcode-11)
 
+## Register Your App with PushKit at Launch Time
```
Comment (idelgado) [inline iOS-13-Migration-Guide.md]: I don't think it makes sense here either. It should be a step under each migration guide.

**Review by idelgado — CHANGES_REQUESTED** [review]

File: iOS-13-Migration-Guide.md (hunk 2 above)
Comment (piyushtank) [reply iOS-13-Migration-Guide.md]: @idelgado addressed the review feedbacks. lmk

**Review by idelgado — APPROVED** [review]

**Pull request merged**